### PR TITLE
EVSS: Add other documents to events timeline

### PR DESCRIPTION
**Pull request by carlmjohnson**
Refs https://github.com/department-of-veterans-affairs/sunsets-team/issues/197

### CODE REVIEW

File: app/serializers/disability_claim_detail_serializer.rb [hunk 1]
```diff
@@ -66,6 +69,16 @@ def create_events_for_tracked_items
     end.flatten
   end
 
+  def create_events_for_documents
+    # Objects with trackedItemId are part of other events, so don't duplicate them
+    docs = sub_objects_of('vbaDocumentList').select { |obj| obj['trackedItemId'].nil? }
+    docs = create_documents docs
+    docs.map do |obj|
+      obj[:type] = :other_documents_list
```
Comment (austin-martinez) [inline app/serializers/disability_claim_detail_serializer.rb]: Does this need a `date` key to be sorted correctly?

File: spec/serializers/disability_claim_detail_serializer_spec.rb [hunk 2]
```diff
@@ -44,4 +44,23 @@
...
+        JSON.parse raw_claim
+      end
+    end
+    let(:disability_claim) do
+      FactoryGirl.build(:disability_claim, data: raw_data)
+    end
+    let(:other_documents_count) do
+      attributes['events_timeline'].count { |obj| obj['type'] == 'other_documents_list' }
+    end
+    it 'should not duplicate documents' do
```
Comment (austin-martinez) [inline spec/serializers/disability_claim_detail_serializer_spec.rb]: Can this description be modified? The test is actually that the documents without tracked item IDs have the correct `type` set, there isn't anything in the code that checks for duplicates

Comment (carlmjohnson) [inline spec/serializers/disability_claim_detail_serializer_spec.rb]: Everything that is pulled out of the vbaDocumentList gets the type other_documents_list, so if more than one if the four items made it through, that would indicate a bug.

File: app/serializers/disability_claim_detail_serializer.rb (hunk 1 above)
Comment (carlmjohnson) [inline app/serializers/disability_claim_detail_serializer.rb]: Good call.

File: spec/serializers/disability_claim_detail_serializer_spec.rb (hunk 2 above)
Comment (austin-martinez) [inline spec/serializers/disability_claim_detail_serializer_spec.rb]: Only documents without tracked item id are being pulled out of that list though
`docs = sub_objects_of('vbaDocumentList').select { |obj| obj['trackedItemId'].nil? }`
I understand what was meant by this description, but the code itself doesn't handle duplicates, it only checks the tracked item id.

**Pull request merged**